spec/scopes-and-launch-context/index.md

```diff
@@ -136,8 +136,7 @@ If an application requests a clinical scope which is restricted to a single pati
 ### Standalone apps
 
 Standalone apps that launch outside the EHR do not have any EHR context at the
-outset. These apps must explicitly request EHR context by using the following
-scopes:
+outset. These apps must explicitly request EHR context. The EHR SHOULD provide the requested context if requested by the following scopes:
```

Comment:
I agree that there may be cases (e.g., if the clinician explicitly decides hit a "skip" button on a patient picker) where the EHR wouldn't supply this context to an app, despite successfully completing the app launch. That said, I think there's a notion of "must support" missing here at least for a couple of context -- e.g., the EHR must support resolving a patient, even if not every launch will take advantage of this.

Reply:
Suggested some alternate wording to better describe your intent as explained above; however, I'd also like to question the basic premise -- why is it important for an EHR to provide a patient picker UI? This is wayyyy outside OAuth.